temperatures/routes.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+from typing import List, Optional
+
+from fastapi import Depends, HTTPException
+from sqlalchemy.orm import Session
+
+from cities.models import DbCity
+from dependencies import app, get_db, fetch_temperature, ListDep
+from temperatures import crud
+from temperatures.models import DbTemperature
+from temperatures.schemas import Temperature
+
+
+@app.post("/temperatures/update")
+async def update_temperatures(db: Session = Depends(get_db)) -> dict[str, str]:
+    cities = db.query(DbCity).all()
+
+    for city in cities:
+        try:
+            temp = await fetch_temperature(city.name)
```

Comment:
The function `fetch_temperature` is awaited, implying it is asynchronous. Ensure that `fetch_temperature` is indeed an async function; otherwise, this will raise an error.